extensions/core/interfaces/collections/interface.vue

```diff
@@ -0,0 +1,46 @@
+<template>
+  <v-select
+    :value="value"
+    :disabled="readonly"
+    :id="name"
+    :options="choices"
+    :placeholder="options.placeholder"
+    @input="$emit('input', $event)" />
+</template>
+
+<script>
+import mixin from "../../../mixins/interface";
+
+export default {
+  mixins: [mixin],
+  computed: {
+    choices() {
+      const collections = this.$store.state.collections || {};
+      let choices = {};
+
+      for (var key in collections) {
+        if (collections.hasOwnProperty(key) && collections[key].collection) {
+          if (this.options.include_system || collections[key].collection.substring(0, 9) != 'directus_'){
```

Comment:
`collections[key].collection.substring(0, 9) != 'directus_'` could also be written as

`collections[key].collection.startsWith("directus_") === false`, which is IMO a little more readable

Reply:
Also, `'directus_'` => `"directus_"` 🤓